src/osd/PrimaryLogPG.cc

```diff
@@ -15,51 +15,41 @@
  *
  */
 
-#include <errno.h>
-
-#include <charconv>
-#include <sstream>
-#include <utility>
```

Comment:
i still think this should belong to this commit.

Reply:
OK - will remove.

Reply:
reinstated these ones

Reply:
this comment is not addressed.